auth-web/src/components/auth/common/Product.vue

```diff
@@ -244,6 +249,7 @@ export default class Product extends Mixins(AccountMixin) {
     let { code } = this.productDetails
     let subTitle = `${code && code.toLowerCase()}CodeSubtitle`
     let details = `${code && code.toLowerCase()}CodeDescription`
+    let note = `${code && code.toLowerCase()}CodeNote`
```

Comment:
would `code?.toLowerCase()` work?

Reply:
Absolutely! will update this and the others 👍

Reply:
Cleans up a lot of code, optionalChaining 🥳 
Short circuit used to be the way before ts brought this in, its the little things in life ya know!